hack/arktos-up-scale-out-poc.sh

```diff
@@ -54,13 +54,17 @@ fi
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-# for POC, the kubelet_flags is used for the new temporary kubelet commandline args
-KUBELET_FLAGS="--tenant-servers="${TENANT_SERVERS}
-# --tenant-servers set the tenant server urls for node-lifecycle controller
-CONTROLLER_MANAGER_EXTRA_FLAGS="--tenant-servers="${TENANT_SERVERS}
 
-echo KUBELET_FLAGS for new kubelet commandline --tenant-servers
-echo ${KUBELET_FLAGS}
+TENANT_SERVERS_KUBELET=${TENANT_SERVERS_KUBELET:-}
```

Comment:
TENANT_SERVERS_KUBELET seems a bit confusing , from the error code it seem it is the server, not the kubelet.

Reply:
This is the --tenant-servers arg used in kubelet. The code was not ready to pick up kubeconfig so I had to use a workaround. This is explained in the manual.